README.md

```diff
@@ -14,13 +14,30 @@ HPE OneView makes it simple to deploy and manage today’s complex hybrid cloud
 
 The HPE OneView Terraform SDK enables developers to easily build integrations and scalable solutions with HPE OneView and HPE Image Streamer.
 
-You can find the latest supported HPE OneView Terraform Provider SDK [here](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v6.6.0-13)
+You can find the latest supported HPE OneView Terraform Provider SDK [here](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v7.0.0-13)
 
 ## What's New
 
-HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 3800 (OneView v6.6)
+HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 4000 (OneView v7.0)
 
 Please refer to [notes](https://github.com/HewlettPackard/terraform-provider-oneview/blob/master/CHANGELOG.md) for more information on the changes , features supported and issues fixed in this version
+### Migration to HPE OneView 7.0 from HPE OneView 6.x ( Non-Synergy appliance )
+With HPE OneView 7.0 one can migrate your server hardware from HPE OneView 6.x using migrate functionality provided in the UI. 
+With server hardware migration , corresponding server profile will also get migrated to HPE OneView 7.0 . 
+
```

Comment:
At the end, may be we can add "For more details one need to check HPE OneView documentation on migration supportability"